rust/signer/src/util.rs

```diff
@@ -14,8 +14,23 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+use tiny_keccak::Keccak;
 use libc::size_t;
 
+pub trait Keccak256<T> {
+	fn keccak256(&self) -> T where T: Sized;
+}
+
+impl Keccak256<[u8; 32]> for [u8] {
+	fn keccak256(&self) -> [u8; 32] {
```

Comment:
I think such function is already part of `tiny_keccak`: https://docs.rs/tiny-keccak/1.4.2/tiny_keccak/fn.keccak256.html